design-documents/graph-ql/coverage/customer/customer.graphqls

```diff
@@ -27,6 +27,7 @@ type Mutation {
     resetPassword(email: String!, resetPasswordToken: String!, newPassword: String!): Boolean  @resolver(class: "\\Magento\\CustomerGraphQl\\Model\\Resolver\\ResetPassword") @doc(description: "Reset a customer's password using the reset password token that the customer received in an email after requesting it using requestPasswordResetEmail.")
 }
 
+# todo: We have a problem here that some fields should be required, but that is a breaking schema change?
 input CustomerAddressInput {
```

Comment:
I've currently left this Todo in place, because the fields shouldn't have been made optional from the start. All input types should be non nullable by default and be made nullable when they actually can be nullable.